test/e2e/framework/pod/wait.go

```diff
@@ -528,7 +528,7 @@ func WaitForPodSuccessInNamespaceSlow(ctx context.Context, c clientset.Interface
 // Unlike `waitForPodTerminatedInNamespace`, the pod's Phase and Reason are ignored. If the pod Get
 // api returns IsNotFound then the wait stops and nil is returned. If the Get api returns an error other
 // than "not found" and that error is final, that error is returned and the wait stops.
-func WaitForPodNotFoundInNamespace(ctx context.Context, c clientset.Interface, podName, ns string, timeout time.Duration) error {
+func WaitForPodNotFoundInNamespace(ctx context.Context, c clientset.Interface, ns, podName string, timeout time.Duration) error {
```

Comment:
There are many places that using `namespace, podName` style. 
- So I fixed this by change the order here.

#113298 refactors the WaitForPodToDisappear and WaitForPodNotFoundInNamespace, they are using different orders. This is why we encounter this.

Reply:
The order is inconsistent.  Let's revert and then fix it to work as before.

Long-term none of the APIs should rely on "correct" ordering, because there is none :cry: Instead, "name + namespace" should get passed through a NamespacedName object. But we can discuss that separately.